doc/source/tests/checkpoint.rst

```diff
@@ -1,37 +1,111 @@
+.. _checkpoint-tests:
+
 ----------------
 Checkpoint Tests
 ----------------
 
-Tests for restart checkpoints. Runs 2D implosion using PPM method with slope mesh refinement. 
+At the time of writing, there are currently two mechanisms for checkpointing:
+
+1. The new approach that writes checkpoints with the ``"check"`` method, that promises a lot more flexibility.
+2. The legacy approach that uses Charm++'s machinery
+
+The intention is to entirely transition to the new approach. But, at this time the new approach has limitations (see `Issue #316 <https://github.com/enzo-project/enzo-e/issues/316>`_). For that reason, both approaches are automatically tested (by the pytest machinery).
```

Comment:
The intention is _not_ to transition to the new approach. Each has its advantages and disadvantages, so both should be kept. Charm++ CR can do lots of things Cello CR cannot.